routes/event.ts

```diff
@@ -72,8 +90,10 @@ function getEventsDataByQuery(req: Request, res: Response): void {
 
   events
     .find(commonQuery, {buyers: false})
+    .limit(limit)
     .lean(true)
-    .exec((err: Error, data: Event[]): void => {
+    .exec()
+    .then((data: Event[]) => {
```

Comment:
data =>

Reply:
If I delete type I have tslint Error `Property 'map' does not exist on type 'Object'`